File: src/MdiTabStrip/MdiTabStripItemBase.cs
using System;
using System.ComponentModel;
using System.Windows.Forms;

namespace MdiTabStrip
{
    [ToolboxItem(false)]
    public partial class MdiTabStripItemBase : Control
    {
        public void InvokeMouseDown(MouseEventArgs e)
        {
            OnMouseDown(e);
        }

        public void InvokeMouseEnter(EventArgs e)
        {
            OnMouseEnter(e);
        }

        public void InvokeMouseHover(EventArgs e)
        {
            OnMouseHover(e);
        }

        public void InvokeMouseLeave(EventArgs e)
        {
            OnMouseLeave(e);
        }

        public void InvokeMouseMove(MouseEventArgs e)
        {
            OnMouseMove(e);
        }

        public void InvokeMouseUp(MouseEventArgs e)
        {
            OnMouseUp(e);
        }
    }
}

File: MdiTabStrip/MdiTabStripItemBase.vb
<System.ComponentModel.ToolboxItem(False)> _
Public Class MdiTabStripItemBase
    Inherits Control

    Public Sub InvokeMouseDown(ByVal e As System.Windows.Forms.MouseEventArgs)
        OnMouseDown(e)
    End Sub

    Public Sub InvokeMouseEnter(ByVal e As System.EventArgs)
        OnMouseEnter(e)
    End Sub

    Public Sub InvokeMouseHover(ByVal e As EventArgs)
        OnMouseHover(e)
    End Sub

    Public Sub InvokeMouseLeave(ByVal e As System.EventArgs)
        OnMouseLeave(e)
    End Sub

    Public Sub InvokeMouseMove(ByVal e As System.Windows.Forms.MouseEventArgs)
        OnMouseMove(e)
    End Sub

    Public Sub InvokeMouseUp(ByVal e As System.Windows.Forms.MouseEventArgs)
        OnMouseUp(e)
    End Sub
End Class
